data/sql/updates/pending_db_world/rev_1626444381466806000.sql

```diff
@@ -0,0 +1,17 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626444381466806000');
+
+DELETE FROM `creature_loot_template` WHERE `Entry` = 193 AND `Item` = 21949;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(193, 21949, 0, 0.5, 0, 1, 1, 1, 1, 'Blue Dragonspawn - Design: Ruby Serpent');
+
+DELETE FROM `creature_loot_template` WHERE `Entry` = 6130 AND `Item` = 21949;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(6130, 21949, 0, 0.5, 0, 1, 1, 1, 1, 'Blue Scalebane - Design: Ruby Serpent');
+
+DELETE FROM `creature_loot_template` WHERE `Entry` = 6129 AND `Item` = 21949;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(6129, 21949, 0, 0.5, 0, 1, 1, 1, 1, 'Draconic Magelord - Design: Ruby Serpent');
+
+DELETE FROM `creature_loot_template` WHERE `Entry` = 6131 AND `Item` = 21949;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(6131, 21949, 0, 0.5, 0, 1, 1, 1, 1, 'Draconic Mageweaver - Design: Ruby Serpent');
```

Comment:
Please rewrite this so that less queries has to be made. You can see https://www.azerothcore.org/wiki/sql-standards#compact-queries if you are unsure how to do it =)

Reply:
Done, I've edited the commit! I hope it's all good, @Azcobu tested it for me and got "Query OK, 4 rows affected (0.00 sec)
Records: 4  Duplicates: 0  Warnings: 0". I guess it's fine? Tell me if I need to change something else :).